packages/explorer/src/app/api/world/utils/deduplicateAbi.ts

```diff
@@ -0,0 +1,29 @@
+import { Abi, AbiItem } from "viem";
+
+function hasNamedInputs(entry: AbiItem) {
+  if (!("inputs" in entry) || !entry.inputs || !entry.inputs.length) {
+    return false;
+  }
+  return entry.inputs.some((input) => input.name);
+}
+
+export function deduplicateAbi(abi: Abi) {
```

Comment:
what's this help with?

Reply:
Discussed in https://github.com/latticexyz/mud/pull/3025